src/cgame/cg_rocket_draw.cpp

```diff
@@ -797,16 +797,6 @@ class SpeedGraphElement : public HudElement
 		currentSpeedElement->SetClass( "speed_current", true );
 	}
 
-	void OnPropertyChange( const Rml::PropertyIdSet& changed_properties ) override
```

Comment:
I don't think it really does any harm to support this, especially since the goal is to have user modifiable HUDs. The real thing is surfacing documentation on each of these custom elements, their attributes, and their styling properties...

Reply:
what the current rcss does is to add the background color on the parent element